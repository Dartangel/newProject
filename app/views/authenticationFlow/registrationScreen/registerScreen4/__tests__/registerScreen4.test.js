import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import RegistrationScreen4 from '../registerScreen4';

describe('RegistrationScreen4', () => {
    test('RegistrationScreen4 snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<RegistrationScreen4/>);
        expect(snapshot).toMatchSnapshot();
    });
});