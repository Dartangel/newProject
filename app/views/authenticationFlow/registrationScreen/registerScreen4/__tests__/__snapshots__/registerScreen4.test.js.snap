// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationScreen4 RegistrationScreen4 snapshot 1`] = `
<ImageBackground
  source={
    Object {
      "testUri": "../../../app/src/img/splashScreen1.jpeg",
    }
  }
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 0.25,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 20,
        }
      }
    >
      У вас есть проблемы с коленями?123
    </Text>
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 15,
          "paddingLeft": 15,
        }
      }
    >
      Выберите тренировку по своему состоянию
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RegisterSettings
      title="Я в порядке"
    />
    <RegisterSettings
      title="Без прыжков"
    />
    <RegisterSettings
      title="Малая ударная нагрузка"
    />
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "green",
          "borderRadius": 50,
          "flex": 0.75,
          "justifyContent": "center",
          "marginHorizontal": 80,
          "marginVertical": 20,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "black",
            "fontSize": 20,
          }
        }
      >
         Cледующие 
      </Text>
    </TouchableOpacity>
  </View>
</ImageBackground>
`;
